### ISSUE TITLE
how to add two Souce Endporint  

### ISSUE THREAD

**Issue by madadimy** [ajainarayanan/react-dag#10]
how to add two Souce Endporint and  two Sink Endporint in type of  'transform'?  [ like this demo](https://jsplumbtoolkit.com/community/demo/flowchart/index.html)

**Comment by ajainarayanan** [comment]
@madadimy I am not sure if I understand your question. Today we have a node type `transform` that will have two endpoints. One will accept incoming connections and one will have connections going out from it. Does your use case require something different? 

In the demo that you have linked each node has 4 endpoints with connections going in and coming out. The existing `transform` type nodes have only 2.

Is your question `Can a transform node have 4 endpoints`?

**Comment by madadimy** [comment]
yes i want konw how to add 4 endpoints in a transform node?  there have two endpoints can accept incoming connections and  two endpoints can connections going out from it.

**Comment by madadimy** [comment]
@ajainarayanan   yes  i want know  how to add 4 endporint in a transform node . there have two endpoints can accept incoming connections and two endpoints can connections going out from it.

**Comment by ajainarayanan** [comment]
@madadimy Sorry for the extremely delayed response. Been busy at work. Right now you can't do that. I need to come up with a design for that. Will update this thread once I have a design and a PR for it.

**Comment by ajainarayanan** [comment]
@madadimy I have re-written react-dag and the new design allows user to add N number of endpoints to a node. You can achieve this using the new design (version 2.0.0.alpha.1) from npm (https://www.npmjs.com/package/react-dag)
  
Please check the README for documentation.

**Comment by rankyangel2014** [comment]
Can you provide the source code？

**Comment by rankyangel2014** [comment]
Can you provide the source code of demo

**Comment by rankyangel2014** [comment]
http://react-dag-app.surge.sh/complexdag

can you show me the source code ？
think you -_- !

**Comment by rankyangel2014** [comment]
http://react-dag-app.surge.sh/complexdag

can you show me the source code ？
thanks   -_- !

**Comment by ajainarayanan** [comment]
@rankyangel2014 Sorry didn't realize I didn't push the branch to github. Here you go https://github.com/ajainarayanan/react-dag-app/tree/feature/ui-2.0.0.alpha.1

**Comment by rankyangel2014** [comment]
thanks a lot
